be/src/exprs/expr.cpp

```diff
@@ -274,7 +276,13 @@ Status Expr::create_vectorized_expr(starrocks::ObjectPool* pool, const starrocks
     }
     case TExprNodeType::COMPUTE_FUNCTION_CALL:
     case TExprNodeType::FUNCTION_CALL: {
-        if (texpr_node.fn.name.function_name == "if") {
+        if (texpr_node.fn.binary_type == TFunctionBinaryType::SRJAR) {
+#ifdef STARROCKS_WITH_HDFS
```

Comment:
> A pthread thread can only attach a Java virtual machine. However, libhdfs will create Java virtual machine in the internal, will do some additional initialization of the operation. If we create the Java virtual machine in advance, then these initialization operations will not be executed, resulting in some errors outside of expectations. So we can not create additional virtual machine, need to reuse the virtual machine in libhdfs.

We can additionally implement the getJNIEnv function later to avoid relying on libhdfs.
But if UDF wants to work with libhdfs, we must rely on the functions in libhdfs